vite-project/src/strategypage/RadarChart.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useRef, useEffect } from "react";
+
+export interface point {
+  x: number;
+  y: number;
+}
+
+export interface RadarInput {
+  name: string;
+  value: number;
+  max: number;
+}
+
+export interface RadarComponentProps {
+  inputs: RadarInput[];
+}
```

Comment:
get width too

Reply:
^^^ and then don't use the polygonRadius parameter everywhere

Reply:
also the canvas parameter is unnecessary if you already have a reference to it. These are private functions so you can just guarantee the reference is always present when they're called